packages/ui/atoms/Figure/FigureTwicpics.ts

```diff
@@ -93,6 +87,18 @@ export class FigureTwicpics<T extends BaseProps = BaseProps> extends Figure<
     return this.formatSrc(super.original);
   }
 
+  /**
+   * Get the current device pixel ratio
+   * Returns 1 if user agent is considered as a bot.
+   */
+  get devicePixelRatio() {
+    if (!this.$options.dpr) {
```

Comment:
**suggestion:** Test for bots here 

This would allow to remove the `if (!isBot)` condition below and would align the behaviour with the docblock description. 

```suggestion
    if (!this.$options.dpr || isBot) {
```